doc/pre_install.rst

```diff
@@ -0,0 +1,70 @@
+.. include:: links.inc
```

Comment:
this whole file is new.  Some of the info is from the "historical note" on the old installation landing page. Some I added (like links to MNE-HCP and MNE-BIDS).  The section about "what should I install?" is probably the part where other sets of eyes are most needed.